sample/github/legit.go

```diff
@@ -21,22 +21,21 @@ import (
 	"encoding/json"
 	"flag"
 	"fmt"
-	"os"
-	"strings"
-
-	"github.com/golang/glog"
+	ghclient "github.com/google/go-github/github"
 	"golang.org/x/oauth2"
-	webhooks "gopkg.in/go-playground/webhooks.v3"
 	"gopkg.in/go-playground/webhooks.v3/github"
-
-	ghclient "github.com/google/go-github/github"
+	"io/ioutil"
+	"log"
+	"net/http"
+	"os"
+	"strings"
 )
 
 const (
 	// Environment variable containing json credentials
 	envSecret = "GITHUB_SECRET"
 	// this is what we tack onto each PR title if not there already
-	titleSuffix = "I buy it"
+	titleSuffix = "looks pretty legit"
 )
 
 // GithubHandler holds necessary objects for communicating with the Github.
```

Comment:
This comment is for line 44: the struct's `ctx` field, but I can't comment on that line since it's outside the diff context.

There's no need for this field. `http.Request.Context()` will return a context to use for the github client requests.

I find it humorous that the [context docs](https://golang.org/pkg/context/) say explicitly, "do not store Contexts inside a struct type," and [`http.Request` does exactly that](https://golang.org/src/net/http/request.go?s=2888:11106#L302). Do as I say, not as I do.  :grin: